test/fixtures/requests/computeLedgerHashTransactions.json

```diff
@@ -1,21 +1,16 @@
 [
   {
-    "hash": "F8F337DEE5D5B238A10AF4A4D56926BA26C83EE7AF5A5A6474340C56F9252DF3",
-    "date": "2015-08-12T01:01:10+00:00",
-    "ledger_index": 15202439,
-    "tx": {
```

Comment:
Why did this format change?

Reply:
So originally, this fixture was in the format returned by `getLedger`. Since we're removing `getLedger()` in favor of `request({command: Ledger})`, which iirc uses a `Ledger` typescript type, we should switch over the ledger hash computation to use the TS type.